java/src/main/native/CMakeLists.txt

```diff
@@ -129,6 +133,7 @@ set(CUDF_INCLUDE "${PROJECT_SOURCE_DIR}/../../../../cpp/include"
 
 find_library(CUDF_LIBRARY "cudf"
     HINTS "$ENV{CUDF_ROOT}/lib"
```

Comment:
Is this necessary? If others are OK I would like to remove this. I don't think we need this to build on the host directly.

Reply:
It follows the conventions of the other libraries. and is where we would find libcudf if it were installed instead of just being built.